app/helpers/form_helper.rb

```diff
@@ -94,4 +94,11 @@ def international_phone_codes_data(code_data)
       country_name: code_data['name'],
     }
   end
+
+  def validated_form_for(record, options = {}, &block)
+    options[:data] ||= {}
+    options[:data][:validate] = true
+    javascript_pack_tag_once('form-validation')
+    validated_form_for(record, options, &block)
```

Comment:
this should be `simple_form_for` .... it's causing infinite recursion/stackoverflows

```suggestion
    simple_form_for(record, options, &block)
```

Reply:
Whoops. Mass search and replace bit me 😅